### DISCUSSION TITLE
关于 Kamol 近期出现的乱加价、中介不负责等行为讨论

### DISCUSSION THREAD

**Discussion by Misaka-0x447f** [project-trans/basket-relay#11] [General]
据 https://t.me/jessicalee85 反馈，最近 Kamol 的中文中介普遍信用不佳。据报告中介 wuyanlii 吴艳丽存在手术前几天从 33 万泰铢加价到 40 万泰铢的行为，且拒绝加价则威胁扣留签证。建议出国就医避开中文中介、谨慎交出签证原件。出于保护当事人原因暂无法提供截图。

**Comment by Misaka-0x447f** [comment]
吴艳丽的不负责行为：手术后 2-3 周接着做 FFS
![image](https://user-images.githubusercontent.com/15797507/222655179-80c10696-3197-44da-a57d-585f2a33f442.png)

**Comment by Neko-aLIEz** [comment]
我们收到了，感谢您的回报
